website/docs/api-backends.md

```diff
@@ -927,6 +927,47 @@ Backpropagate the Swish activation
 | `inplace`   | <tt>bool</tt>     | If `True`, the `dY` array is modified in place. |
 | **RETURNS** | <tt>FloatsXd</tt> | The gradient of the input.                      |
 
+### Ops.dish {#dish tag="method" new="8.1.1"}
+
+<inline-list>
+
+- **default:** <i name="yes"></i>
+- **numpy:** <i name="no"></i>
+- **cupy:** <i name="yes"></i>
+
+</inline-list>
+
+Dish or "Daniël's Swish-like activation" is an activation function with a
+similar shape to Swish or GELU. However, Dish does not rely on elementary
```

Comment:
Maybe hyperlink to Swish and GELU and say that "with a similar non-monotonic shape" and its like Swish, GELU and also Mish.

Reply:
Changed the wording similar to what you suggested and added links to GELU, Swish and Mish.